aws_lambda_powertools/utilities/advanced_parser/schemas/sqs.py

```diff
@@ -0,0 +1,61 @@
+import re
+from datetime import datetime
+from typing import Dict, List, Optional
+
+from pydantic import BaseModel, root_validator, validator
+from typing_extensions import Literal
+
+
+class SqsAttributesSchema(BaseModel):
+    ApproximateReceiveCount: str
+    ApproximateFirstReceiveTimestamp: datetime
+    MessageDeduplicationId: Optional[str]
+    MessageGroupId: Optional[str]
+    SenderId: str
+    SentTimestamp: datetime
+    SequenceNumber: Optional[str]
+    AWSTraceHeader: Optional[str]
+
+
+class SqsMsgAttributeSchema(BaseModel):
+    stringValue: Optional[str]
+    binaryValue: Optional[str]
+    stringListValues: List[str] = []
+    binaryListValues: List[str] = []
+    dataType: str
+
+    @validator("dataType")
+    def valid_type(cls, v):  # noqa: VNE001
```

Comment:
docs: could you add a comment as a context for this validation?

Reply:
added